src/new/wallet.js

```diff
@@ -153,10 +154,16 @@ class HathorWallet extends EventEmitter {
     }
 
     if (storage) {
+      /**
+       * @type {import('../types').IStorage}
```

Comment:
Since we are importing the `IStorage` I think we can remove the import here and on the one below

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-lib/pull/613/commits/99b5968fe84fa4e8175111fc1026eb22b88ff413